rustler/src/alloc.rs

```diff
@@ -0,0 +1,46 @@
+use std::alloc::{GlobalAlloc, Layout};
+
+const SIZEOF_USIZE: usize = std::mem::size_of::<usize>();
+const MAX_ALIGN: usize = 8;
```

Comment:
Is this the alignment that ERTS wants? Not sure how to interpret this.

Reply:
This is the alignment that ERTS /gives/. The docs only say "aligned enough for all builtin types", the largest builtin type is `double` at 8 bytes.